### DISCUSSION TITLE
Can't rewind/fast forward on Android

### DISCUSSION THREAD

**Discussion by Godd67** [advplyr/audiobookshelf-app#645] [Q&A]
Thanks for the great app!
Looks like the only way now I can change listening position within a chapter or on global progress bar backwards and forward is by clicking Jump buttons.  If I just tap on a progress bar, I see it registers my touch with a small circle but position not adjusted. Web player works as expected. Is it a known issue? I swear it worked before.
My phone is Galaxy S21.

**Comment by hobesman** [comment]
I think you have to tap on the dot itself, not just the progress bar generally.

I know there was discussion about changing this but I don't know off the top of my head what it was originally or what it was (possibly) changed to.

**Comment by advplyr** [comment]
This was updated so that you have to drag the cursor on the track. The problem was people were often accidently touching the progress bar and losing their position.

**Comment by Godd67** [comment]
Hmm, I can't make it work. What is an exact action sequence? 
I tried just drag my finger along progress bar, tap and drag, tap, hold and drag - no go.

**Reply by Godd67** [reply]
Ok, figured it out - you have to be very precise to hit that circle, almost impossible to do it with my thumb while walking or on the train  :)
